pkgs/development/python-modules/numpy/1.nix

```diff
@@ -94,6 +94,7 @@ buildPythonPackage rec {
       --replace 'py.full_path()' "'python'"
 
     substituteInPlace pyproject.toml \
+      --replace-fail "Cython>=0.29.34,<3.1" Cython \
```

Comment:
Any reason why the Cython replacement is not quoted? I mean i suppose it works, but was this intended to be unquoted?

Reply:
fair enough :)